src/vs/editor/browser/controller/mouseTarget.ts

```diff
@@ -404,26 +408,53 @@ abstract class BareHitTestRequest {
 
 class HitTestRequest extends BareHitTestRequest {
 	private readonly _ctx: HitTestContext;
-	public readonly target: HTMLElement | null;
-	public readonly targetPath: Uint8Array;
+	private readonly _eventTarget: HTMLElement | null;
+	public readonly hitTestResult = new Lazy(() => MouseTargetFactory.doHitTest(this._ctx, this));
+	private _useHitTestTarget: boolean;
+	private _targetPathCacheElement: HTMLElement | null = null;
+	private _targetPathCacheValue: Uint8Array = new Uint8Array(0);
+
+	public get target(): HTMLElement | null {
+		if (this._useHitTestTarget) {
+			return this.hitTestResult.value.hitTarget;
+		}
+		return this._eventTarget;
```

Comment:
I think `_useHitTestTarget` is not needed:
```suggestion
		return this._eventTarget ?? this.hitTestResult.value.hitTarget;
```

Reply:
Oh, nevermind, `_useHitTestTarget` is changed later.